recipes/geff-spec/recipe.yaml

```
@@ -0,0 +1,48 @@
+context:
+  name: geff-spec
+  version: "1.1.0.post0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/geff_spec-${{ version }}.tar.gz
+  sha256: 1b265c7849d4937307f83e12cbebabdd20af02967a065603ce7b8c6dc11e0290
```

```suggestion
  sha256: 2ecb96da9af89d3d1f69e1849964c5bfbfd45dcf7374411b5c91541d9e8a2595
```
@tlambert03 sorry must have copied the wrong hash. This one should be correct!